src/3045R5_quantities_and_units_library.md

```diff
@@ -4467,7 +4467,7 @@ library does not have enough information to print it that way by itself.
 5. `std::chrono::duration` uses 'Q' and 'q' for a number and a unit. In the grammar above, we
    proposed using 'N' and 'U' for them, respectively. We also introduced 'D' for dimensions. Are
    we OK with this?
-6. Are we OK with the usage of '_' for denoting a subscript identifier? Should we use it everywhere
+6. Are we OK with the usage of `_` for denoting a subscript identifier? Should we use it everywhere
```

Comment:
With single quotes, I wanted to express that I mean a character, as was the case for `Q` and `q` in the previous point. Maybe we should use `'_'` instead everywhere?

Reply:
Done.